server/server.go

```diff
@@ -586,7 +587,13 @@ func (s *Server) wipeIfAppropriate(ctx context.Context, imagesToDelete []string)
 			}
 		}
 		for _, sb := range s.ContainerServer.ListSandboxes() {
-			if err := s.removePodSandbox(ctx, sb); err != nil {
+			const timeoutSec = 0
+
+			if err := s.stopPodSandbox(ctx, sb, timeoutSec); err != nil {
```

Comment:
I would not expect any containers to be running in this case. We should have already figured out it's a reboot scenerio

Reply:
Should we avoid calling `s.restore` in case of a wipe: https://github.com/cri-o/cri-o/blob/5a46e7eb35333c4d4bb71e7d470b71236cea67ac/server/server.go#L451

Reply:
hmm it would break image wiping if we didn't know which containers previously were running. plus, we need to be able to call into CNI to remove the networking resources, which restore sets up

Reply:
Hm, this is tricky so I do not want to change the logic too much to break existing setups. The linked bug (https://issues.redhat.com/browse/OCPBUGS-3183) indicates that there are containers running which _maybe got restored_? The timeout of 10 seconds will make CRI-O wipe slow, especially since we do not stop them in parallel.

Reply:
Are we missing the state that tells us whether it is a reboot vs. a crio restart? If we detect a reboot we could take a big hammer and all of the state w/o having to go through individual pods.